.github/ISSUE_TEMPLATE/release-checklist.md

```diff
@@ -371,8 +373,6 @@ Notes
 [func-prow-badge]: https://prow.knative.dev/badge.svg?jobs=release_func_main_periodic
 [func-prow-job]: https://prow.knative.dev?job=release_func_main_periodic
 
-[operator-version-badge]: https://img.shields.io/github/release-pre/knative/operator.svg?sort=semver
```

Comment:
This seems to be dropped?

Reply:
It's a dupe - it's already present on line 403